utl/flatbuffer/txt_to_flat.cc

```diff
@@ -0,0 +1,97 @@
+// Copyright (c) by respective owners including Yahoo!, Microsoft, and
+// individual contributors. All rights reserved. Released under a BSD (revised)
+// license as described in the file LICENSE.
+
+#ifdef _WIN32
+#define NOMINMAX
+#include <WinSock2.h>
+#else
+#include <sys/socket.h>
+#include <arpa/inet.h>
+#endif
+#include <sys/timeb.h>
+#include "../../vowpalwabbit/parse_args.h"
+#include "../../vowpalwabbit/parse_regressor.h"
+#include "../../vowpalwabbit/accumulate.h"
+#include "../../vowpalwabbit/best_constant.h"
+#include "../../vowpalwabbit/vw_exception.h"
+#include <fstream>
+
+#include "../../vowpalwabbit/vw.h"
```

Comment:
Any files that have already been included in `vw_to_flat.h` which is in turn included here don't need to be included again (e.g. `vw.h`)